04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -134,10 +137,11 @@ $ mkdir models
 $ touch models/student.rb
 ```
 
-> The `app.rb` file is our main application file. This is where alot of the main program logic will live.
+> The `app.rb` file is our main application file. This is where a lot of the main program logic will live.
 
 > The `Gemfile` will contain all the dependencies for our program.
-The `models/student.rb` file will contain the class definition for the Student class that will represent the students table in SQL
+
+> The `models/student.rb` file will contain the class definition for the Student class that will represent the students table in SQL
 
 > by convention we always name our model file names singular
```

Comment:
i know you mentioned whiteboarding the conventions - might be useful to put a table of all the conventions you plan on going over at the bottom for quick future reference